parser/src/test/java/com/microsoft/gctoolkit/parser/test/diary/G1JVMConfigurationTest.java

```diff
@@ -193,4 +193,33 @@ public void testForDetailsAdaptiveSizeRSet() {
         }
     }
 
+    @Test
+    public void testUnifiedG1GC() {
```

Comment:
What is it testing?

Reply:
Unfied G1 logs???

Reply:
The test doesn't describe what part of the program it's testing and there's no assert. This reads like a run() command...

Reply:
It's testing the state of the JVM Configuration after parsing a unified G1 log.. and there are asserts being called.. dozens of them.

Reply:
Oh, so testWith has the asserts, got it, thanks!